sdk/videoanalyzer/azure-media-videoanalyzer-edge/src/samples/README.md

```diff
@@ -0,0 +1,202 @@
+# Azure Video Analyzer Edge client library for Java
```

Comment:
The format for README in the samples directory is slightly different.  We don't want to have the same documentation in two places as it requires changing in two places if we need to update something. Instead, this file should only describe the samples in `src/samples/com/azure/media/videoanalyzer/edge/` directory and all other sections just point to the top-level README.

Take a look at this for example - https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/appconfiguration/azure-data-appconfiguration/src/samples/README.md